src/app/features/room-nav/RoomNavUser.tsx

```diff
@@ -0,0 +1,49 @@
+import { Avatar, Box, Icon, Icons } from 'folds';
```

Comment:
Improve call member display
```suggestion
import { Avatar, Box, Icon, Icons, Text } from 'folds';
```